ShowerShow/Repository/Interface/IScheduleRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ShowerShow.DTO;
+using ShowerShow.Models;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ShowerShow.Repository.Interface
+{
+    public interface IScheduleRepository
+    {
+        public Task CreateSchedule(CreateScheduleDTO schedule, Guid id);
+        public Task<List<Schedule>> GetAllSchedules(Guid UserId);
```

Comment:
Same with List->Ienumerable stuff from the get story you have bro its a simple change but i dont think you need to change it specifically here, changing it in one branch is fine since it will overwrite it